app/utils.py

```diff
@@ -8,4 +11,15 @@ def verify_password(plain_password, hashed_password):
 
 
 def get_password_hash(password):
-    return pwd_context.hash(password)
\ No newline at end of file
+    return pwd_context.hash(password)
+
+
+def create_access_token(data: dict, expires_delta: timedelta | None = None):
+    to_encode = data.copy()
+    if expires_delta:
+        expire = datetime.now(timezone.utc) + expires_delta
+    else:
+        expire = datetime.now(timezone.utc) + timedelta(minutes=15)
```

Comment:
Why are we adding 15 as a constant here when we already have ACCESS_TOKEN_EXPIRE_MINUTES?